build.gradle.kts

```diff
@@ -129,7 +129,7 @@ tasks {
       rule {
         limit {
           counter = "BRANCH"
-          minimum = BigDecimal(0.89)
+          minimum = BigDecimal(0.88)
```

Comment:
I did get rid of some BS tests..

Reply:
I am very happy to see the collaborators test go. Even so this is the second time this value has gone down, and whilst last time it was probably a rounding error this time this time there are three uncovered branches
 
 https://1965-327621961-gh.circle-artifacts.com/0/coverage/uk.gov.justice.digital.hmpps.hmppstier.service/index.html
 
 vs
 
 https://1965-327621961-gh.circle-artifacts.com/0/coverage/uk.gov.justice.digital.hmpps.hmppstier.service/index.html
 
 Specifically the newly uncovered branches are here, somewhat to my surprise 
 https://1967-327621961-gh.circle-artifacts.com/0/coverage/uk.gov.justice.digital.hmpps.hmppstier.service/TierCalculationService.kt.html#L44

Reply:
I'll look at these.